server/lib/notifications.js

```diff
@@ -27,7 +27,7 @@ module.exports = (Sequelize, activity) => {
             activity.type
           ],
           GroupId: activity.GroupId,
-          channel: ['gitter', 'slack', 'twitter'],
+          channel: ['gitter', 'slack', 'twitter', 'email'],
```

Comment:
good catch. I wanted to refactor the emailLib and add that through the notification library. I'll do that separately.